src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
No, if it exists in the Project.JSON then it will be read from there; if it does not exist in the project.json then we will try to see if the file exists